net.sourceforge.Pdfedit.metainfo.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should be upstreamed.

Reply:
I really would like to leave the metainfo.xml and the .desktop file here.

This is a legacy qt3 project, I don't fully understand its old qt3 build system (I had to use some ugly build patches to get the project to build) and afaik installing a .desktop file is not just copy the file but needs also a call to update applications database, and the application won't benefit or use that at all as it's not being packaged in any distro (and when it was packaged for Fedora it was the fedora package that added the .desktop file).

Hopefully this is not a blocker for you.